Sources/NIO/ByteBuffer-builder.swift

```diff
@@ -0,0 +1,123 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2017-2018 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Dispatch
+
+// MARK: - Serializable
+
+/// Used to write a conforming type into a `ByteBuffer` so that it can be used in a result builder.
+public protocol ByteBufferSerializable {
```

Comment:
unprefixed global protocol

Reply:
Why does it need a prefix?

Reply:
https://github.com/apple/swift-nio/blob/main/docs/public-api.md#promises-the-nio-team-make